ui/components/multichain/pages/permissions-page/permissions-page.scss

```diff
@@ -1,9 +0,0 @@
-.permissions-page {
-  &__tabs {
-    padding: 0 16px;
-
-    .tab {
-      flex-grow: 1;
-    }
-  }
-}
```

Comment:
None of these styles are used as far as I can tell

- `permissions-page__tabs`

<img width="314" height="268" alt="Screenshot 2025-09-18 at 2 10 55 PM" src="https://github.com/user-attachments/assets/02d792bf-b958-452d-a464-7c34281fcf29" />